### PR TITLE
Renata/fix small models

### PR DESCRIPTION
**Pull request by r-akemii**
In this PR:
* Updated zone builder (refactored part of the code)
* Added TQDM progress bars when downloading population files
* Added a new verification to import_political_subdivisions to avoid useless Nominatim requests and writing repeated lines in the database
* Added a function to retrieve the model_place's country when we re-open a Tradesman model

### CODE REVIEW

**Comment by r-akemii** [comment]
@pedrocamargo although we have made some considerable changes in the zone building, the existing unittests are working fine. I'll create a new branch to migrate all tests to pytest before moving on to something new.

**Comment by r-akemii** [comment]
@pedrocamargo can you help me with this test fail later? It's breaking in the build documentation part and I'm not sure why..

**Comment by pedrocamargo** [comment]
The pinning of the Sphinx version is due to https://github.com/pydata/pydata-sphinx-theme/issues/1098 
It can be removed once this issue is resolved.

File: tradesman/model.py [hunk 1]
```diff
@@ -186,6 +187,14 @@ def __initialize_model(self):
             self._project.new(self.__folder)
             add_new_tables(self._project.conn)
 
+    def __country_identifier(self):
+
+        country = self._project.conn.execute(
```
Comment (pedrocamargo) [inline tradesman/model.py:192]: If the query is empty, this will fail, since you cannot take the first element of it.

Comment (r-akemii) [reply tradesman/model.py]: I've made some changes in the code to overcome this issue. Now I'm saving some project information using AequilibraE's About module. It probably will be less error-prone.

**Pull request merged**